5_evaluate.R

```diff
@@ -2,13 +2,20 @@ source('5_evaluate/src/eval_utility_fxns.R')
 source('4_visualize/src/plot_data_utility_fxns.R')
 
 p5 <- list(
+  # For now, evaluate only predictions for lakes within CASC states
+  tar_target(p5_CASC_states,
+             c('ND','SD','IA','MI','IN','IL','WI','MN','MO','AR','OH')),
```

Comment:
Is this also being applied to what we are bundling up somewhere?

Reply:
No, not at the moment. Do we want to restrict that as well?

Reply:
At the moment, NLDAS output is zipped by state, so is easy for the user to partition if they only want results for CASC states. The GCM output for now is zipped by tile, so is less simple to partition to CASC states.

Reply:
OK, just thinking about when we go to upload how there is a bit of a manual step right now